lib/tdf3/src/utils/index.ts

```diff
@@ -278,3 +282,28 @@ function base64clean(str: string) {
 export function base64ToBytes(str: string) {
   return toByteArray(base64clean(str));
 }
+
+/**
+ *
+ * Function generates key, it returned both KeyForEncryption and KeyForManifest.
+ *   `KeyForEncryption === KeyForManifest` produces true;
+ *
+ * @returns {Object}:
+ * {
+ *   keyForEncryption: Binary;
+ *   keyForManifest: Binary;
+ * }
+ */
+export async function keyMiddleware() : Promise<{
+  keyForEncryption: KeyInfo; keyForManifest: KeyInfo;
+}>{
+  const tdfService = TDF.create({ cryptoService: WebCryptoService });
+  tdfService.setEncryption({ type: 'split' });
+  if (!tdfService.encryptionInformation?.generateKey) {
+    throw new Error('Crypto service not initialised')
+  }
+
+  const key1 = await tdfService.encryptionInformation.generateKey();
+
+  return { keyForEncryption: key1, keyForManifest: key1 };
```

Comment:
Isn't the default using the same key for both?

Reply:
Yep but its default middleware is here
https://github.com/virtru/virtru-tdf3-js/pull/586/files#diff-5631231d382b7c1a4ac85b4c2a941793890a0f5f12348a232a76e2fd3b45ae89R99

and set as default middlware. https://github.com/opentdf/client-web/pull/241/files#diff-549436fb06c43a51e130655ea4292b8cedd3a77dd8364603423f4fe3cc5b3865R360

Or Virtru-sdk has two usecases, CKS and RCA which case its always different key